docs/sources/administration/api-keys/create-api-key.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Create an API key in Grafana
+menuTitle: Create an API key
+description: 'How to create an API key in Grafana'
+weight: 50
+keywords:
+  - API keys
+  - Service accounts
+---
+
+# Create an API key in Grafana
+
+Create an API key when you want to manage your computed workload with a user that is not associated with a user.
```

Comment:
@eleijonmarck just seeing this now, it reads funny "manage your workload with a user that is not associated with a user."

Reply:
great catch